src/hooks/useQuery.ts

```diff
@@ -33,7 +35,7 @@ export const useQuery = <T = any, V = object>(
   args: UseQueryArgs<V>
 ): UseQueryResponse<T> => {
   const unsubscribe = useRef(noop);
-  const client = useClient();
+  const client = useClient(args.client);
```

Comment:
Oh before I forget 😅 let’s get this added to the other hooks as well.

`useSubscription` will just need another arg, just like `useQuery`. For `useMutation` we can probably use an optional second argument.